### ISSUE TITLE
Switch.System.Net.DontEnableSchUseStrongCrypto and Switch.System.Net.DontEnableSystemDefaultTlsVersions not working in web.config

### ISSUE THREAD

**Issue by raych1** [dotnet/docs#7106]
It works when I set them in app.config for console application. However, it doesn't work when set them in web.config for website project. 
This is the settings I add.
```
<runtime>
    <AppContextSwitchOverrides value="Switch.System.Net.DontEnableSchUseStrongCrypto=false;Switch.System.Net.DontEnableSystemDefaultTlsVersions=false"/>
</runtime>
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e59350ee-7837-34f8-b50e-c9c136720155
* Version Independent ID: 19747ce7-da26-bf2e-5066-1df11e61b0d0
* Content: [&lt;AppContextSwitchOverrides&gt; Element](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element)
* Content Source: [docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md](https://github.com/dotnet/docs/blob/master/docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md)
* Product: **dotnet-framework**
* GitHub Login: @rpetrusha
* Microsoft Alias: **ronpet**

**Comment by raych1** [comment]
Related to [Microsoft/dotnet#816](https://github.com/Microsoft/dotnet/issues/816)

**Comment by BillWagner** [comment]
this was fixed in #7789
